static/bidder-info/resetdigital.yaml

```diff
@@ -0,0 +1,21 @@
+adapters:
+  resetdigital:
+    endpoint: http://b-us-east14.resetdigital.co:9001
+    meta-info:
+      maintainer-email: biddersupport@resetdigital.co
+      app-media-types:
+        - banner
+        - video
+        - audio
+      site-media-types:
+        - banner
+        - video
+        - audio
+      supported-vendors:
+      vendor-id: 1162
+    usersync:
+      cookie-family-name: resetdigital
+      redirect:
+        url: https://sync.resetdigital.co/csync?pid=rubicon&redir={{redirect_url}}
+        support-cors: false
+        uid-macro: '$USER_ID'
```

Comment:
shape of yaml seems to be incorrect. for example,  no need to include `adapters` `meta-info`  `app-media-types` `site-media-types` key, maintainer email is nested incorrectly etc.

Refer following as example: https://github.com/prebid/prebid-server/blob/06ac44bbca993745ffd4158c40f9042430fcbde2/static/bidder-info/appnexus.yaml#L1-L21
```
endpoint: "http://ib.adnxs.com/openrtb2"
platform_id: "5"
maintainer:
  email: "prebid@microsoft.com"
gvlVendorID: 32
capabilities:
  app:
    mediaTypes:
      - banner
      - video
      - native
  site:
    mediaTypes:
      - banner
      - video
      - native
userSync:
  key: "adnxs"
  redirect:
    url: "https://ib.adnxs.com/getuid?{{.RedirectURL}}"
    userMacro: "$UID"
```